linode/login_client.py

```diff
@@ -219,6 +291,28 @@ def _login_uri(self, path):
         return "{}{}".format(self.base_url, path)
 
     def generate_login_url(self, scopes=None, redirect_uri=None):
+        """
+        Generates a url to send users so that they may authenticate to this
```

Comment:
the description, and function name, calls it a "url" but the params call it a "uri". Inconsistent and should be fixed if possible.

Reply:
The spec calls it a "redirect uri," hence the naming here, but this function returns a URL (always pointing to https://login.linode.com)..so I'm going to keep it even if it's a little weird.